package.json

```diff
@@ -60,7 +61,8 @@
     "lint:quiet": "npm run lint -- --quiet",
     "eject": "react-scripts eject",
     "storybook": "start-storybook -p 9009 -s public",
-    "build-storybook": "build-storybook -s public"
+    "build-storybook": "build-storybook -s public",
+    "swagger:generate-schema": "dtsgen -o ./src/__generated__/apiSchema.d.ts -n API --url https://acc.api.top.amsterdam.nl/api/v1/swagger/?format=openapi"
```

Comment:
Also this needs to be added as a build step in Jenkins. And README for development.

Reply:
:+1: